gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -82,6 +192,65 @@ message FeedHeader {
   extensions 9000 to 9999;
 }
 
+message Shape{
+  required string shape_id = 1;
+  required ShapePoint shape_point = 2;
+}
+
+message ShapePoint{
+  required float shape_pt_lat = 1;
+  required float shape_pt_lon = 2;
+  optional float shape_dist_traveled = 3;
+}
+
+message Route{
+  required string route_id = 1;
+  required string agency_id = 2;
+  optional TranslatedString route_short_name = 3;
+  optional TranslatedString route_long_name = 4;
+  optional TranslatedString route_desc = 5;
+  optional RouteType route_type = 6;
```

Comment:
`route_type` should be `required`

Reply:
Also, if it's required, then we don't need to worry about specifying a default. If it is optional, then we do (and we'd need another `UKNOWN` value as the default).